gtsfm/utils/graph.py

```diff
@@ -55,8 +55,7 @@ def prune_to_largest_connected_component(
         Subset of unit_translations which are in the largest connected components.
     """
     input_edges = [k for (k, v) in rotations.items() if v is not None]
-    # TODO(Ayush): start using pose priors by uncommenting the following line
-    # input_edges += [k for (k, v) in pose_priors.items() if v is not None]
+    input_edges += [k for (k, v) in pose_priors.items() if v is not None]
```

Comment:
and also remove the line above?

Reply:
what above further below, do we use rotations or pose_priors?

Reply:
This filters the rotations and unit translations down to the largest connected component. The priors are used to add the extra edges when figuring out the connected components. 

For example, the upward facing camera is not connected to any other camera in the pairs given to the two view estimator. But we have strong priors on cam2 to other cams in the same timestamp and hence we should not throw away the connected component of cam2s.